youtube_dl/extractor/nuevo.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import float_or_none
+
+class NuevoBaseIE(InfoExtractor):
+    def _extract_nuevo(self, config_url, video_id, info=None, ignore_hd=False):
+        if info is None:
+            info = {}
+
+        sdformats, hdformats = [], []
+        tree = self._download_xml(config_url, video_id, transform_source=lambda s: s.strip())
+
+        for child in tree:
```

Comment:
Instead of this loop use [anitube's approach](https://github.com/nexAkari/youtube-dl/blob/6722ebd43720e836af8217fa078fa1a604b98229/youtube_dl/extractor/anitube.py#L35-L51). For text fields use `utils.xpath_text`.